projpicker/web/web.py

```diff
@@ -0,0 +1,170 @@
+#!/bin/env python3
+import http.server
+import json
+import argparse
+import projpicker as ppik
```

Comment:
```python
import argparse
import webserver
import json

import projpicker as ppik
```
Standard modules first, blank link, third-party modules (none), blank line, our modules. Internally, I use the logical order of modules (which module is used first and later?).